copyright

```diff
@@ -2165,6 +2165,41 @@ Files:
 Copyright: RisingLeaf (https://github.com/RisingLeaf)
 License: CC-BY-SA-4.0
 
+Files:
+ images/scene/volcanic-eruption-at-night.jpg
+Copyright: None; public domain. Creator Vide/A.Hubner, US National Park Service (https://www.nps.gov/media/video/view.htm?id=2F152320-6D7D-4CF1-875D-37B3F3216293)
+License: None; public domain.
```

Comment:
```suggestion
Files:
 images/scene/volcanic-eruption-at-night*
Copyright: N/A (public domain)
License: public-domain
Comment:
 Taken from https://archive.is/I6L1t. From the US National Park Service, and therefore in the public domain because it was created by government employees while doing work for the government.
```